docs/source/examples/update_dataset.rst

```diff
@@ -0,0 +1,57 @@
+#################
+ Update Dataset
+#################
+
+This topic describes how to update dataset, including:
+
+- :ref:`examples/update_dataset:Update Label`
+
+This examples assumes that you already uploaded a dataset to tensorbay.
+If you haven't done so already, See :ref:`features/dataset_management:Upload Dataset`
+
+There is a scenario with update dataset
+
+# Upload dataset and commit as version 1.0.
+# Update label on dataset and commit as version 2.0.
+# Add data to dataset and commit as version 3.0.
+# ...
+
+See the following detailed instructions.
+
+***************
+ Update Label
+***************
+
+TensorBay SDK supports methods to update label. You can upload label to overwrite previous label.
+
+Before overwrite label you need get the previously uploaded dataset and create a draft:
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Update label / get dataset an create draft"""
+   :end-before: """"""
+
+If catelog changed, update it:
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Update label / update catalog"""
+   :end-before: """"""
+
+Overwrite previous label with new label on dataset:
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Update label / overwrite label"""
+   :end-before: """"""
+
+Commit dataset:
+
+.. literalinclude:: ../../../examples/update_dataset.py
+   :language: python
+   :start-after: """Update label / commit dataset"""
+   :end-before: """"""
+
+| Now dataset is committed with a version by new label.
+| You can switch different commits to use previous and new label.
+
```

Comment:
Write an IMPORTANT here to demonstrate that if new types of labels are added, and a user wants to keep the previous labels in the new version, both of these labels should be included in the updated data.